packages/mdc-floating-label/mdc-floating-label.scss

```diff
@@ -37,7 +37,13 @@
       transform $mdc-floating-label-transition-duration $mdc-animation-standard-curve-timing-function,
       color $mdc-floating-label-transition-duration $mdc-animation-standard-curve-timing-function;
     line-height: 1.15rem;
+    text-overflow: ellipsis;
+    white-space: nowrap;
     cursor: text;
+    overflow: hidden;
+    // Force the label into its own layer to prevent to prevent visible layer promotion adjustments
```

Comment:
"to prevent" is repeated

Also, this sounds like something that should be a separate fix, not in this PR...

Reply:
I can create a PR for that.